### DISCUSSION TITLE
Add the function to switch languages

### DISCUSSION THREAD

**Discussion by muzier9** [greenelab/lab-website-template#280] [Ideas] [answered]
For our lab, I wanted to write this website to let applicants see our group in detail. Since our lab is in China, there are not only foreign researchers, but also many Chinese researchers visiting this website, so I want to add a function to switch languages, which is written on the header or footer. Click the drop-down box to select Chinese or English. Is there a good and simple way to realize this function?

**Comment by vincerubinetti** [comment, marked as answer]
Unfortunately this is something the template doesn't support out-of-the-box. See this issue: https://github.com/greenelab/lab-website-template/issues/5

My recommendation is to check out this jekyll plugin:
https://github.com/kurtsson/jekyll-multiple-languages-plugin

And add it to your template by following these instructions:
https://greene-lab.gitbook.io/lab-website-template-docs/advanced/jekyll-plugins
